src/ucp/proto/proto_common.c

```diff
@@ -283,7 +286,8 @@ ucp_proto_common_calc_completion(const ucp_proto_common_init_params_t *params,
     ucp_proto_perf_range_t *range =
             &params->super.caps->ranges[params->super.caps->num_ranges++];
 
-    if (perf_params->is_multi) {
+    if (perf_params->is_multi ||
+        (params->flags & UCP_PROTO_COMMON_INIT_FLAG_HDR_ONLY)) {
```

Comment:
can header be bigger than single fragment?

Reply:
fixed (in another commit)